Configurations/common0.tmpl

```diff
@@ -11,14 +11,14 @@
      map { $_ => 1 }
      ( # The sources of stuff may be generated
          ( map { @{$unified_info{sources}->{$_}} }
-               keys %{$unified_info{sources}} ),
+               sort keys %{$unified_info{sources}} ),
          $disabled{shared}
              ? ()
              : ( map { @{$unified_info{shared_sources}->{$_}} }
-                 keys %{$unified_info{shared_sources}} ),
+                 sort keys %{$unified_info{shared_sources}} ),
          # Things we explicitly depend on are usually generated
          ( map { $_ eq "" ? () : @{$unified_info{depends}->{$_}} }
-               keys %{$unified_info{depends}} ));
+               sort keys %{$unified_info{depends}} ));
```

Comment:
Because this whole expression generates a new hash table, sorting of keys inside it is unnecessary.